andres-uzeda/homework7/mvc_task/src/Controller.java

```diff
@@ -0,0 +1,40 @@
+import javax.swing.JOptionPane;
+public class Controller {
```

Comment:
This names are generic use something as UserController or UserModel for your classes